packages/sources/src/components/tree-view/tree-view.models.ts

```diff
@@ -1,18 +1,22 @@
-export interface TreeViewItem<T> {
+export interface TreeViewItem {
   /** The reference of type T */
-  reference: T;
+  reference: string;
```

Comment:
Zouden we 'm ondertussen niet beter "id" kunnen noemen? Dan dwingt de naam ook al wat unieks af

Reply:
Het item "bouwwerken" is net als "complexen" een item waar de onderliggende items worden bijgeladen. Voor "bouwwerken" is een vertraging van 250 ms ingesteld waardoor dit niet heel erg opvalt. Als je "bouwwerken" eenmaal hebt opengeklapt, werkt het zoeken op "bouw" weer als verwacht. 

De `reference` property is nu `id` geworden.